.eslintrc

```
@@ -0,0 +1,38 @@
+{
+    "root": true,
+    "env": {
+        "browser": true,
+        "node": true,
+        "es6": true,
+        "es2018": true,
+        "jest": true
+    },
+    "extends": [
+        "airbnb-base",
+        "airbnb-typescript/base",
+        "plugin:prettier/recommended"
+    ],
+    "plugins": ["prettier", "@typescript-eslint"],
+    "globals": {
+        "process": "readonly"
+    },
+    "rules": {
+        "prettier/prettier": "warn",
+        "prefer-const": ["warn"],
+        "no-console": ["error", { "allow": ["warn", "error", "info"] }],
+        "curly": "warn",
+        "no-undef": "off",
+        "eol-last": ["error", "always"],
+        "max-len": ["error", { "code": 120 }],
+        "no-return-await": [
+            "error"
+        ],
+        "import/prefer-default-export": "off",
+        "no-restricted-syntax": "off"
+    },
+    "ignorePatterns": ["out/**", "*.js"],
```

Either replace with dist or remove
```suggestion
    "ignorePatterns": ["dist/**", "*.js"],
```